tests/treeTest.py

```diff
@@ -0,0 +1,19 @@
+from coalahtml.tree import Tree
+import unittest
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        self.FL = [
+                    "/home/tushar/coala-html/tests",
+                    "/home/tushar/coala-html/coalahtml",
+                    "/home/tushar/coala-html/tests/specs/directives/",
+                    "/home/tushar/coala-html/setup.py",
+                    "/home/tushar/coala-html/tests/TestUtilities.py"
+                  ]
+
+    def test_graph(self):
+        tr = Tree(self.FL)
+        testChild = tr.root.child['tests'].child.get('specs', None)
+        self.assertTrue(testChild)
```

Comment:
That will be handled too.
As I said each node is being identified with two attributes:
* parent of that node
* id
These two attributes makes any node unique.
No two nodes can have same set of the two attributes.